### PR TITLE
Update apt error logging

### PR DESCRIPTION
**Pull request by dheyay**
Fixes: #3299

## Why is this needed?

This PR solves all of our problems because...

## Test Steps

---

- [ ] *(un)check this to re-run the checklist action*

### CODE REVIEW

**Review by orndorffgrant — CHANGES_REQUESTED** [review]
As part of our process to get ready for v35, I have rebased `next-v35` onto `main`. As a result, `next-v35` is now obsolete. Please rebase this on `main` and target the PR to `main`.

**Comment by dheyay** [comment]
Rebased on  main @orndorffgrant

**Review by orndorffgrant — COMMENTED** [review]
From the checklist

> - [x] I've properly referenced all bugs that this PR fixes

The bug is not properly referenced in the commit message.

Otherwise, looks great!

File: uaclient/apt.py [hunk 1]
```diff
@@ -377,6 +379,7 @@ def run_apt_update_command(
             cmd=["apt-get", "update"], override_env_vars=override_env_vars
         )
     except exceptions.APTProcessConflictError:
+        LOG.warning("Error running apt update")
```
Comment (lucasmoura) [inline uaclient/apt.py:382]: Should we capture the exception and log it here too ?

File: uaclient/apt.py [hunk 2]
```diff
@@ -448,6 +452,7 @@ def run_apt_install_command(
             override_env_vars=override_env_vars,
         )
     except exceptions.APTProcessConflictError:
```
Comment (lucasmoura) [inline uaclient/apt.py:454]: Same suggestion here

**Review by lucasmoura — APPROVED** [review]
Just two nits, but LGTM

File: uaclient/apt.py (hunk 2 above)
Comment (dheyay) [reply uaclient/apt.py:454]: I think this one is covered by the logging on line `455`

File: uaclient/apt.py (hunk 1 above)
Comment (dheyay) [reply uaclient/apt.py:382]: Good catch! Will update this one

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**